sympy/functions/combinatorial/factorials.py

```diff
@@ -174,7 +174,7 @@ def _eval_is_positive(self):
 
     def _eval_is_real(self):
         if ((self.args[0].is_negative and self.args[0].is_noninteger) or
-           self.args[0].is_nonnegative):
+                self.args[0].is_nonnegative):
```

Comment:
I've explained it in [this](https://github.com/sympy/sympy/pull/8727#discussion_r22399641) comment.